### ISSUE TITLE
Feature Request: use the symfony route name to group the transactions

### ISSUE THREAD

**Issue by Ardenexal** [getsentry/sentry-symfony#547]
In the Performance view for Symfony, routes are listed per URL instead of transaction name which makes it more difficult to get to the overall picture of a route

In javascript, we have the ability to add the transaction name to group transactions together. Could this functionality be extended to Symfony as well?

**Comment by ste93cry** [comment]
>Could this functionality be extended to Symfony as well?

Yes, absolutely. What it's not clear to me is if the feature of the JS SDK is to just set the name of the transaction to the name of the route or if there is another hidden feature I'm not aware of that allows to group multiple transactions together 🤔 Can you please shed some light on it, maybe pointing me to the code that does it?

**Comment by stayallive** [comment]
The `setName` on the transaction is what is being grouped on. This should be the same value as the `transaction` string on an event so that events can be linked to transactions.

[In Laravel](https://github.com/getsentry/sentry-laravel/blob/c9077d6765bff3f7f8ed7e91774d5bd224847a2b/src/Sentry/Laravel/Integration.php#L126-L168) we try to generate a route name as generic as possible (and we also do this for the transaction field on events).

**Comment by odoucet** [comment]
I'm bumping this thread as I face the same issue.
I was able to rewrite the transaction name in getTracesSampler() through `$context->getTransactionContext()->setName()` but it seems to be ignored (the unmodified transaction name is sent to Sentry).

**Comment by odoucet** [comment]
FYI I was able to rewrite transaction name by listening to `kernel.response` and rewrite it with : 
```
    $transaction = \Sentry\SentrySdk::getCurrentHub()->getTransaction();

        if ($transaction !== null && $transaction->getSampled()) {
            $transaction->setName('REWRITE HERE');
        }
```

**Comment by cleptric** [comment]
I'll close this as it's a duplicate of #649 
We'll go ahead with the route name.
